ConsoleApp1/HomeWork/Weather.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1.HomeWork
+{
+    public class Weather
+    {
+        public async Task RunAsync()
+        {
+            WeatherCenter weatherCenter = new WeatherCenter();
+
+            weatherCenter.TemperatureChanged += temp => { Console.WriteLine($"Temperature: {temp}"); }; ;
```

Comment:
can you please stop copy/paste and start thinking at every sign? what reason do you add an extra semicolon at the end?